### PR TITLE
[5962] Edit user dialog - edit password stuck after error

### PR DESCRIPTION
**Pull request by jvega190**
- Fix edit password stuck after error
- Add submitting state in EditGroupDialog
- Update the Groups and Users form in accordance to changes with PATCH /groups and /users #5964

https://github.com/craftercms/craftercms/issues/5962
https://github.com/craftercms/craftercms/issues/5964

### CODE REVIEW

File: ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx [hunk 1]
```diff
@@ -211,11 +212,11 @@ export function EditGroupDialogUI(props: GroupEditDialogUIProps) {
             {!group.externallyManaged && (
               <div className={classes.formActions}>
                 {isEdit && (
-                  <SecondaryButton disabled={!isDirty} onClick={onCancel}>
+                  <SecondaryButton disabled={!isDirty || inProgress} onClick={onCancel}>
                     <FormattedMessage id="words.cancel" defaultMessage="Cancel" />
                   </SecondaryButton>
                 )}
-                <PrimaryButton disabled={!submitOk} type="submit">
+                <PrimaryButton disabled={!submitOk || inProgress} loading={inProgress} type="submit">
```
Comment (rart) [inline ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx]: Doesn't the `loading` prop disable already?
```suggestion
                <PrimaryButton disabled={!submitOk} loading={inProgress} type="submit">
```

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/EditGroupDialog/EditGroupDialogUI.tsx]: it does, updated

**Comment by jvega190** [comment]
Pending to move to ready for review - waiting for https://github.com/craftercms/craftercms/issues/5966

File: ui/app/src/components/EditGroupDialog/EditGroupDialogContainer.tsx [hunk 2]
```diff
@@ -332,6 +349,7 @@ export function EditGroupDialogContainer(props: EditGroupDialogContainerProps) {
       onFetchMoreUsers={fetchMoreUsers}
       hasMoreUsers={usersHaveNextPage}
       disableAddMembers={disableAddMembers}
+      inProgress={isSubmitting}
```
Comment (rart) [inline ui/app/src/components/EditGroupDialog/EditGroupDialogContainer.tsx]: inProgress means is submitting? Why call it differently?

**Review by rart — APPROVED** [review]

**Pull request merged**